lib/modules/platform/gitlab/index.ts

```diff
@@ -697,7 +697,9 @@ export function massageMarkdown(input: string): string {
   let desc = input
     .replace(regEx(/Pull Request/g), 'Merge Request')
     .replace(regEx(/PR/g), 'MR')
-    .replace(regEx(/\]\(\.\.\/pull\//g), '](!');
+    .replace(regEx(/\]\(\.\.\/pull\//g), '](!')
+    // Strip unicode null characters as GitLab markdown does not permit them
+    .replace(regEx(/\u0000/g), "");
```

Comment:
I can add this each platforms' versions of `massageMarkdown` 👍 GitHub at least doesn't seem to need it, but I don't think it will do any harm.

I'm probably not the right person to create a function that's shared across all of them though without a few more pointers about where to put it - still getting across the codebase.

Reply:
I'm ok with merging this as is and deferring the rest.

Right now we don't have a convenient location for massaging all which I'm aware of